projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-users-endpoint.php

```diff
@@ -38,6 +38,7 @@
 			'search'          => '(string) Find matching users.',
 			'search_columns'  => "(array) Specify which columns to check for matching users. Can be any of 'ID', 'user_login', 'user_email', 'user_url', 'user_nicename', and 'display_name'. Only works when combined with `search` parameter.",
 			'role'            => '(string) Specify a specific user role to fetch.',
+			'capabilities'    => '(array) Specify a specific capability to fetch.',
```

Comment:
Since we can pass multiple capabilities, maybe we can change it to plural? Something like `Specify one or more capabilities to fetch.` What do you think?